offlineimap/accounts.py

```diff
@@ -273,6 +273,17 @@ def syncrunner(self):
             if e.severity >= OfflineImapError.ERROR.CRITICAL:
                 raise
             return
+        
+        if self.utf_8_support and self.remoterepos.getdecodefoldernames():
+            import sys, logging
+            logging.error("Configuration mismatch in account '%s'. "% self.getname() +
+                          "\nAccount setting 'utf8foldernames' and repository " +
+                          "setting 'decodefoldernames'\nmay not be used at the " +
+                          "same time. This account has not been synchronized.\n" +
+                          "Please check the configuration and documentation.")
```

Comment:
Use `self.ui`.

Reply:
(instead of logging.error)

Reply:
Actually, I'm not sure self.ui is the correct way to go because this likely won't set the correct exit code.
I think it's better to raise OfflineImapError, level REPO; See offlineimap/error.py.

Reply:
I'm not sure if it's correctly done, but b70f053a2fb8813fdd79475e11dd7d5bb9bf33f4 tries to fix the issue.